easybuild/easyconfigs/n/netcdf4-python/netcdf4-python-1.5.5.1-fosscuda-2020b.eb

```diff
@@ -0,0 +1,55 @@
+# Author: Denis Krišťák (INUITS)
+
+easyblock = 'PythonBundle'
+
+name = 'netcdf4-python'
+version = '1.5.5.1'
+
+homepage = 'https://unidata.github.io/netcdf4-python/'
+description = """Python/numpy interface to netCDF."""
+
+toolchain = {'name': 'fosscuda', 'version': '2020b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/Unidata/netcdf4-python/archive/']
```

Comment:
why did you add this line? the source url for netcdf4-python is already provided in the exts_list.

Reply:
fixed in e5836d9